modules/sdk/src/main/scala/org/tessellation/sdk/domain/cluster/programs/PeerDiscovery.scala

```diff
@@ -3,12 +3,15 @@ package org.tessellation.sdk.domain.cluster.programs
 import cats.effect.{Async, Ref}
```

Comment:
All changes in this file will be discarded. I added these logs for the debug purposes only and they slipped in to the final commit.